src/lib/compilation/package-lib.js

```diff
@@ -18,15 +18,15 @@ module.exports = (pathToProject, entry, baseEslintPath, options = {}) => {
   const outputDirectoryPath = os.tmpdir();
   const outputFilename = `./${libName}.js`;
 
-  const compiler = webpack([{
+  const compiler = webpack({
     mode: 'production',
     entry,
     output: {
       path: outputDirectoryPath,
       filename: outputFilename,
       libraryTarget: options.libraryTarget ? 'umd' : undefined,
       globalObject: options.libraryTarget,
-      hashFunction: 'sha512' // remove when we upgrade to webpack 5
+      hashFunction: 'xxhash64'
```

Comment:
Thanks for leaving your findings [there](https://github.com/medic/cht-conf/pull/540#discussion_r1119345390) @jkuester :raised_hands: